pkg/controllers/schema/schemas.go

```diff
@@ -32,31 +31,23 @@ var (
 	}
 )
 
-type SchemasHandlerFunc func(schemas *schema2.Collection, changedSchemas map[string]*types.APISchema, deletedSomething bool) error
+type SchemasHandlerFunc func(schemas *schema2.Collection) error
 
-func (s SchemasHandlerFunc) OnSchemas(schemas *schema2.Collection, changedSchemas map[string]*types.APISchema, forceChange bool) error {
-	return s(schemas, changedSchemas, forceChange)
+func (s SchemasHandlerFunc) OnSchemas(schemas *schema2.Collection) error {
+	return s(schemas)
 }
 
 type handler struct {
 	sync.Mutex
 
-	// refreshLock prevents refreshAll to be run in parallel
-	refreshLock sync.Mutex
-
-	ctx               context.Context
-	toSync            int32
-	schemas           *schema2.Collection
-	client            discovery.DiscoveryInterface
-	cols              *common.DynamicColumns
-	crdClient         apiextcontrollerv1.CustomResourceDefinitionClient
-	ssar              authorizationv1client.SelfSubjectAccessReviewInterface
-	handler           SchemasHandlerFunc
-	changedIDs        map[k8sapimachineryschema.GroupVersionKind]bool
-	createdCRDs       map[k8sapimachineryschema.GroupVersionKind]bool
-	deletedCRDs       map[k8sapimachineryschema.GroupVersionKind]bool
-	apiServiceChanged bool
-	gvksFromKeys      map[string][]k8sapimachineryschema.GroupVersionKind
+	ctx     context.Context
+	toSync  int32
+	schemas *schema2.Collection
+	client  discovery.DiscoveryInterface
+	cols    *common.DynamicColumns
+	crd     apiextcontrollerv1.CustomResourceDefinitionClient
```

Comment:
I prefer the old name `crdClient` for an object that's a client and not an instance of a CRD

Reply:
Sure, fixed :+1: